decentraland/explorer
# fix: avatars instantiation hiccups

This PR has:
- Kernel: bots.create() & bots.remove() functionality to be used in the browser console to instantiate fake local (no comms) avatars
- Kernel: new URL param "LOS" to control the scenes loading line of sight radius
- Animations initialization id fix:  We found a performance hit when initializing the animations with an ID different than the animation files, so we corrected those animation file names.
- Depmaps caching improvement (developed by Santi in #1080 )

Test instructions for Unity Editor:

0. Open profiler and focus in CPU module
1. In unity, check “debug scenes” in SceneController gameobject (debug scene coords should be far from 0,0 to be able to test in a more isolated way).
2. In WSSController gameobject, check “Use Local comms” and “use custom content server”. Use LOCAL in Environment and use coords 0,0.
3. Get the package with all wearables in asset bundles (ask Pravus or Brian) and unzip it.
4. Open a terminal and go to the directory where the wearables where unzipped and run `npx serve@6.5.8 --cors -p 1338`
5. Open `RendereableAssetLoadHelper.cs`and change `loadingType` to `LoadingType.ASSET_BUNDLE_ONLY`
6. Hit Play in Unity and after the rendering activates, wait for the friends initialization to avoid mixing its impact in CPU with avatars instantiation.
7. After waiting for a bit, you can create local bots from the browser console with `bots.create()` and see the instantiation performance impact in Unity's profiler
